Knative Install on OpenShift (#1520)
knative/docs: docs/install/Knative-with-OpenShift.md

type: "docs"
---

# Knative Install on OpenShift

This guide walks you through the installation of the latest version of [Knative

> Serving](https://github.com/knative/serving) on [OpenShift](https://github.com/openshift/origin) by using the Knative Serving Operator. The operator is available on the OpenShift OperatorHub. After you install Knative Serving, create and deploy an image of a sample "hello world" app onto the newly created Knative cluster.

Reviewer: ```suggestion
Serving](https://github.com/knative/serving) on [OpenShift](https://github.com/openshift/origin) by using the Knative Serving Operator. The operator is available on the OpenShift OperatorHub. After you install Knative Serving, this guide walks you through creating and deploying an image of a sample "Hello World" app onto the new Knative cluster.
```

You can find [guides for other platforms here](./README.md).

## Before you begin

* An OpenShift 4 cluster is required for installation. Visit [try.openshift.com](try.openshift.com) for information on setting up a cluster. You will need cluster administrator privileges to install and use Knative on an OpenShift cluster. 

## Installing the Knative Serving Operator

> 1. Go to **Catalog > OperatorHub** in the OpenShift Web Console. A list of operators for OpenShift, provided by Red Hat as well as a community of partners and open-source projects is provided. Click on the **Knative Serving Operator** tile.

Reviewer: Make "OpenShift Web Console" into a link to the appropriate place.

> > **NOTE:** Use the **Filter by Keyword** box to facilitate the search of the Knative Serving operator in the catalog.

Reviewer: ```suggestion
> **NOTE:** Use the **Filter by Keyword** box to help you find the Knative Serving operator in the catalog.
```

![KSO Tile](images/knative_serving_tile_highlighted.png)

> 2. A **Show Community Operator** dialog box will open. Click **Continue** to proceed.

Reviewer: I know the automatic numbering is not always working out correctly on the website, but we do want to stick to using 1. for each numbered item in a list. At some point the automatic numbering will be fixed, and we won't have to go back and update a lot of static numbers.
